resources/css/style.css

```diff
@@ -405,11 +405,159 @@ footer p {
 }
 
 
+/*-----------------------------------------*/
+/* SECTION DASHBOARD */
+/*-----------------------------------------*/
+
+
+.modal-dialog-dashboard {
+	max-width: 95%;
+}
+
+.modal-content-dashboard {
+	width: 100%;
+}
+
+.close-dashboard {
+	position: relative;
+	left: 100%;
```

Comment:
Is this for keeping the `.close-dashboard` div on the right side? If yes, then I think it's better to use `float: right` and then clear after :D

Reply:
That left:100% property was a bad idea, i was testing something about this one but i forgot to change it. I've put the div on the right side by adding the bootstrap class "justify-content-end" to that div.